Popularity/Kipras/pyspark/rollOnSpark.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
```

Comment:
But these scripts are doing exactly that. 
 - rollOnSpark.sh does the local data transform and starts run.sh
 - run.sh moves data to HDFS and starts SPARK job (pyspark_ml.py)
 - pyspark_ml.py do ML on SPARK and return output
 - run.sh transform output
 - rollOnSpark.sh measure accuracy and plot result